internal/metrics/otelmetrics/timer.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package otelmetrics
+
+import (
+	"context"
+	"time"
+
+	"go.opentelemetry.io/otel/metric"
+)
+
+type otelTimer struct {
+	histogram metric.Float64Histogram
+	fixedCtx  context.Context
+	option    metric.RecordOption
+}
+
+func (t *otelTimer) Record(d time.Duration) {
+	t.histogram.Record(t.fixedCtx, d.Seconds(), t.option)
```

Comment:
Is there a link to OTEL docs that confirms that seconds is the right unit to use?

Reply:
No, I went with the standard unit and it returns a float64 value. I am not sure if Seconds is the right unit to use.

Reply:
https://opentelemetry.io/docs/languages/go/instrumentation/#using-histograms
In this they are using duration.Seconds() for histogram.Record()